Update rest:switch for body_state parameter

quielb opened the pull request:
Update for new parameter body_state.

**Description:**
Update doc for new parameter that sends data in body of get on status update of rest switch

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#26096

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html


<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/10193"><img src="https://gitpod.io/api/apps/github/pbs/github.com/quielb/home-assistant.io.git/f47348fe36750e5fb1a4e6dbd44ea9a2d5a3b917.svg" /></a>

quielb committed:
Update rest:switch for body_state parameter

Update for new parameter body_state.

klaasnicolaas commented on source/_components/switch.rest.markdown:
```diff
@@ -51,6 +51,12 @@ body_off:
+  description: "Allow for data in the body of the status get.  Some IoT devices don't return data when the end point is hit.  They require some query in the body of the get.  This value can be a [template](/topics/templating/)."
+  required: false
+  type: string
+  default: None
```
Is there a default in the code?

quielb commented:
There is no default in the code.  If the option isn't defined then the parameter in the code should have a value of None.  The websession.get allows for the data parameter to be None.

klaasnicolaas commented:
in that case, I think we can remove the `default` variable

quielb committed:
remove default parameter for body_state

quielb commented:
removed

quielb committed:
Update Default

Had to make adjustments to the code because it turns out the bytes function doesn't take a None object.  Set the default to empty string
